certified-connectors/Desk365/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Desk365
```

Comment:
@Desk365-vdpk Please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).